testsuites/lbt-plutus/lbt-plutus-typescript/.envrc

```diff
@@ -0,0 +1,2 @@
+nix_direnv_watch_file build.json
+use flake ../../..#lbt-plutus-typescript
```

Comment:
Hmm i don't think that's a dev shell, should be dev-lbt-plutus-typescript or smtn?

Reply:
This should be solved in flake-lang first, created an issue: https://github.com/mlabs-haskell/flake-lang.nix/issues/39